helix-core/src/test/java/org/apache/helix/controller/rebalancer/waged/model/TestAssignableNode.java

```diff
@@ -200,4 +200,17 @@ public void testParseFaultZone() throws IOException {
 
     Assert.assertEquals(assignableNode.getFaultZone(), "2/testInstance/");
   }
+
+  @Test
+  public void testDefaultInstanceCapacity() {
+    ClusterConfig testClusterConfig = new ClusterConfig("testClusterConfigId");
+    testClusterConfig.setDefaultInstanceCapacityMap(_capacityDataMap);
+
+    InstanceConfig testInstanceConfig = new InstanceConfig("testInstanceConfigId");
+
+    AssignableNode assignableNode =
+        new AssignableNode(testClusterConfig, testInstanceConfig, _testInstanceId,
+            Collections.emptyList());
+    Assert.assertEquals(assignableNode.getMaxCapacity(), _capacityDataMap);
```

Comment:
Why getMaxCapacity? shouldn't you get default capacity instead?

Reply:
The capacity is applied to "max capacity", that is also true when the default capacity is used.